zeeguu_api/api/utils/abort_handling.py

```diff
@@ -0,0 +1,9 @@
+from flask import jsonify
+
+
+def make_error(status_code, message,):
+    response = jsonify({
+        'message': message,
+    })
+    response.status_code = status_code
+    return response
```

Comment:
always newline after last char in file.

Reply:
Is there any specific reason for doing this? I (normally) always do this but I never stopped a think why actually.

Reply:
newline is the official line terminator, and it's nice to terminate all the lines, even the last :)